python/tests/scripts/test_update_storage.py

```diff
@@ -111,6 +120,19 @@ def test_upgrade_script_s3_rbac_ok(moto_s3_endpoint_and_credentials, monkeypatch
     assert s3_storage.credential_key == USE_AWS_CRED_PROVIDERS_TOKEN
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="Test fixture issue with creds")
```

Comment:
Why are we skipping windows here 😢 

If this doesn't work on Windows it doesn't inspire too much confidence in the correctness of the Windows build?

If this is an existing test fixture issue we don't need to fix this in this PR, but these tests really should be running on Windows as well at some point.

Reply:
It's copied from the previous test so I thought the skip is already reviewed and approved....

Reply:
Will do further test on Windows